AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/IncomingAsyncSubscriptionEventPublisher.swift

```diff
@@ -42,7 +42,10 @@ final class IncomingAsyncSubscriptionEventPublisher: Cancellable {
     private let incomingSubscriptionEvents: PassthroughSubject<Event, DataStoreError>
     private let awsAuthService: AWSAuthServiceBehavior
 
-    init(modelSchema: ModelSchema, api: APICategoryGraphQLBehavior, auth: AuthCategoryBehavior?,
+    init(modelSchema: ModelSchema,
+         api: APICategoryGraphQLBehavior,
+         configuration: DataStoreConfiguration,
+         auth: AuthCategoryBehavior?,
```

Comment:
Technically, adding DataStoreConfiguration to the IncomingAsyncSubscriptionEventPublisher is not needed, but will be when we add optimizations to use the equality operator for filtering.